src/content/docs/infrastructure/host-integrations/host-integrations-list/ansible-controller-integration.mdx

```diff
@@ -0,0 +1,180 @@
+---
+title: Red Hat Ansible Automation Controller integration
+tags:
+   - Ansible controller  integration
+   - New Relic integrations
+metaDescription: Install our Ansible Automation Controller dashboards and see your Ansible data in New Relic.
+---
+
+import infrastructureAnsibleController from 'images/infrastructure_screenshot-full_ansible-controller.png'
+
+With our Red Hat Ansible Automation Platform dashboards, you can easily track your Ansible Automation Platform logs, keep an eye on your jobs and monitor instance memory.
```

Comment:
Can we add the word 'visualise' along with easily track?

Reply:
done!